### PR TITLE
Add Iliad testnet and Update Euphoria addresses

### PR DESCRIPTION
**Pull request by hoanm**
### Description

* fix: update latest addresses, and metadata of Euphoria chain and tAURA token

### CODE REVIEW

File: chains/iliad/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  - apiUrl: https://testnet.storyscan.xyz/api
+    family: blockscout
+    name: StoryScan
+    url: https://testnet.storyscan.xyz
+blocks:
+  confirmations: 1
+  estimateBlockTime: 4
+  reorgPeriod: 0
+chainId: 1513
+displayName: Iliad Testnet
```
Comment (paulbalaji) [inline chains/iliad/metadata.yaml:12]: could you also add deployer details to iliad + euphoria metadata please 🙏 

Example here:
https://github.com/hyperlane-xyz/hyperlane-registry/blob/2247762c1b8f96e5a37cbf69481293dc0d84f035/chains/ancient8/metadata.yaml#L12-L14

File: chains/euphoria/addresses.yaml
Comment (paulbalaji) [inline chains/euphoria/addresses.yaml:1]: realised this is a testnet - could you rename and update all references `euphoria` -> `euphoriatestnet` please

File: chains/iliad/metadata.yaml
Comment (paulbalaji) [inline chains/iliad/metadata.yaml:1]: also testnet - please rename and update all references `iliad` -> `storytestnet`

File: chains/iliad/metadata.yaml (hunk 1 above)
Comment (hoanm) [reply chains/iliad/metadata.yaml]: updated

File: chains/euphoria/addresses.yaml
Comment (hoanm) [reply chains/euphoria/addresses.yaml:1]: updated

File: chains/iliad/metadata.yaml
Comment (hoanm) [reply chains/iliad/metadata.yaml:1]: I have removed iliad because Story Testnet existed in the latest pull

**Review by paulbalaji — COMMENTED** [review]
I think you'll want to update euphoria/iliad references in the eAURA/tAURA warp route configs/addresses too

**Comment by hoanm** [comment]
> I think you'll want to update euphoria/iliad references in the eAURA/tAURA warp route configs/addresses too

I updated them. Thanks

**Comment by hoanm** [comment]
> please run `yarn lint:fix`
> 
> could you also combine all changesets into a single `minor` changeset containing the changes in this PR please

Updated

**Comment by paulbalaji** [comment]
> Updated

Thanks! Configs look ok, but I still see multiple changesets, please delete extraneous changesets
image

**Comment by hoanm** [comment]
> > Updated
> 
> Thanks! Configs look ok, but I still see multiple changesets, please delete extraneous changesets image

I removed them ^^

**Review by paulbalaji — APPROVED** [review]

**Pull request merged**